.github/workflows/ci.yml

```
@@ -163,7 +163,7 @@ jobs:
         python-version: '3.11'
     - name: Install build and test dependencies from PyPI
       run: |
-        python -m pip install pytest-xdist -r array-api-tests/requirements.txt
+        python -m pip install -r array-api-tests/requirements.txt
     - name: Build and install Sparse
       run: |
         python -m pip install '.[finch]'
```

> I also moved pytest-xdist to the test dependencies, WDYT?

That won't work, unless we do this too.

```suggestion
        python -m pip install '.[finch,tests]'
```

---

Ah, right. On the second thought let's drop these changes as they're not relevant to `finch-tensor` update and leave it for "light CI refactor" PR.

---

That sounds like a good idea to keep changes isolated.